src/components/Form/formFunctions.ts

```diff
@@ -69,12 +70,15 @@ export const getErrorType = (
     input: string,
     required?: boolean,
     regex?: RegExp,
-    additionalCondition?: (value: string) => boolean
+    additionalCondition?: (value: string) => boolean,
+    deliveryInstructionsOverflow?: boolean
 ): Errors => {
     if (input == "") {
         return required ? Errors.required : Errors.none;
     }
-
+    if (deliveryInstructionsOverflow) {
+        return Errors.deliveryInstructions;
+    }
```

Comment:
Is there a way we can do this generically?

Reply:
I've moved the character limit condition into additionalConditions, but also adding a argument "key" when getting error type, so that I can specify the error message